glib/src/param_spec.rs

```diff
@@ -46,9 +46,10 @@ unsafe impl<'a> crate::value::FromValue<'a> for ParamSpec {
     type Checker = crate::value::GenericValueTypeOrNoneChecker<Self>;
 
     unsafe fn from_value(value: &'a crate::Value) -> Self {
-        let ptr = gobject_ffi::g_value_dup_param(value.to_glib_none().0);
+        let ptr: *mut gobject_ffi::GParamSpec =
+            gobject_ffi::g_value_dup_param(value.to_glib_none().0) as *mut _;
```

Comment:
What is the problem here? Generally we want to put the type on the cast instead of having a wildcard cast and a type annotation.